src/Controls/src/Core/Handlers/Items/iOS/TemplatedCell.cs

```diff
@@ -141,11 +145,12 @@ protected void Layout(CGSize constraints)
 			var rectangle = platformView.Frame.ToRectangle();
 			PlatformHandler.VirtualView.Arrange(rectangle);
 			_size = rectangle.Size;
+			_measureInvalidated = false;
 		}
 
 		public override void PrepareForReuse()
 		{
-			Unbind();
+			_bound = false;
```

Comment:
Don t we need to call UnBind to clear the BindingContext etc?

Reply:
From what I've understood this method is being invoked right before a cell is being reused to fill a different position (i.e. scrolling), so it will be bound to a new view model very soon: there's no need to trigger binding context changed twice (CV2 doesn't have this too).
The disposal of this cell will happen in `CellDisplayingEndedFromDelegate` which will unbind this cell.

Apple’s Documentation states that `prepareForReuse()` is called before an object is returned from the reuse queue, not just when a cell goes offscreen. This implies that cells going offscreen without being dequeued again will not trigger `prepareForReuse()`.